homeassistant/components/hassio/system_health.py

```diff
@@ -57,6 +64,9 @@ async def system_health_info(hass: HomeAssistant) -> dict[str, Any]:
         "disk_used": f"{host_info.get('disk_used')} GB",
         "healthy": healthy,
         "supported": supported,
+        "host_connectivity": network_info.get("host_internet"),
```

Comment:
Let's add Supervisor internet connection info here too. You've already added `supervisor_internet` in the test below, so I assume you had in mind?